IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -392,17 +392,26 @@ - (void)refreshAccessToken:(MSIDBaseToken<MSIDRefreshableToken> *)refreshToken
 
     [self.requestParameters.authority loadOpenIdMetadataWithContext:self.requestParameters
                                                     completionBlock:^(__unused MSIDOpenIdProviderMetadata * _Nullable metadata, NSError * _Nullable error) {
-
-                                                        if (error)
-                                                        {
-                                                            completionBlock(nil, error);
-                                                            return;
-                                                        }
-
-                                                        [self acquireTokenWithRefreshTokenImpl:refreshToken
-                                                                               completionBlock:completionBlock];
-
-                                                    }];
+        
+        if (error)
+        {
```

Comment:
Do we need any log here?

Reply:
Error is logged when error is created using `MSIDCreateError`. Since nothing special here, so no extra log is added :)